frontend/catalyst/api_extensions/error_mitigation.py

```diff
@@ -90,6 +90,10 @@ def mitigated_circuit(args, n):
             s = jax.numpy.array([1, 2, 3])
             return mitigate_with_zne(circuit, scale_factors=s)(args, n)
     """
+
+    if folding.upper() == "RANDOM":
```

Comment:
Instead of introducing another check here, you can modify the one at L151

Reply:
Done.